addons/l10n_ec/migrations/3.4/post-migration.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+
+from odoo import api, SUPERUSER_ID
+
+def update_withhold_type(env):
+    # reclasifies withhold taxes into independent tax groups for sales and purchases
+    env.cr.execute('''
+        update account_tax
+        set tax_group_id=t.id from (select id from account_tax_group where l10n_ec_type='withhold_vat_sale') as t
+        where account_tax.id in (select id from account_tax where tax_group_id in (select id from account_tax_group where l10n_ec_type='withhold_vat') and type_tax_use='sale')
```

Comment:
@jco-odoo I think this one doesn't need any other change. If the module was installed and the CoA was not installed, then the SQL will find no record to affect, because there will be no tax on the old withholding types.